packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -1334,7 +1334,9 @@ describe('Collection aggregations tab', function () {
         return (await activeStage.getText()) === 'Stage 2: select';
       });
 
-      await browser.keys('Escape');
+      const closeButton = await browser.$(Selectors.FocusModeCloseModalButton);
+      await closeButton.click();
```

Comment:
Why not the usual clickVisible on the selector?

Reply:
I can try changing it and see if it still passes 👍

Reply:
Done in https://github.com/mongodb-js/compass/pull/5400/commits/d4ee88377361c1c85ace4c133b1e9910f7c4b08d